device/mellanox/x86_64-mlnx_msn2700-r0/01_mlnx_2700_syslog.conf

```diff
@@ -0,0 +1,6 @@
+# This rsyslog configuration is intended to resolve the following error message that only appears on the MSN2700 platform:
```

Comment:
Do we need a copy right header for this?

Reply:
No, ngci tool will ignore .conf file for copyright header